src/lib/utils/strategies/FlagUnorderedStrategy.ts

```diff
@@ -0,0 +1,47 @@
+import type { UnorderedStrategy } from 'lexure';
+
+export class FlagStrategy implements UnorderedStrategy {
+	public readonly flags: ReadonlySet<string>;
+	public readonly optionHyphens = ['—', '--'];
+	public readonly flagHyphens = ['—', '-'];
+	private readonly kHyphenRegex = /(?:-|—){1,2}/;
```

Comment:
```suggestion
	private readonly hyphenRegex = /(?:-|—){1,2}/;
```

We don't need `k` prefixes in Sapphire. It was Klasa stuff. Don't take the suggestion because it will break other code. Use VSCode refactor instead.

Reply:
`k` prefixes were Skyra. Klasa's lint rules are for ESLint 4.x, they never got updated except to fix removed rules.